src/cookbooks/data_quality/collections.md

```diff
@@ -0,0 +1,26 @@
+## Collections in Bigeye
+Collections help you organize and focus on specific areas of interest, making it simpler to track and address data quality across different segments of your data landscape. This feature enhances efficiency by allowing users to monitor grouped entities in a cohesive manner.
+
+![](../../assets/Bigeye_screenshots/Bigeye-Collections.png)
+
+### Creating a new collection
+If you don’t find a collection that meets your product or requirements, admins can create a new collection.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YTOOTFw5MLw?si=X3VybaWasts-sdjw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+If you're not an admin, please submit a Jira ticket with the necessary details [`here`](https://mozilla-hub.atlassian.net/browse/DENG-4726)
```

Comment:
`here` is not a good click target (it's small and non-descriptive). (it's also not code, so no need for backticks).
This would be better as:

```suggestion
If you're not an admin, please [submit a Jira ticket](https://mozilla-hub.atlassian.net/browse/DENG-4726) with the necessary details.
```